_maps/RandomZLevels/snowdin.dmm

```diff
@@ -530,7 +530,7 @@
 /turf/open/floor/plating,
 /area/awaymission/snowdin/post/dorm)
 "bP" = (
-/obj/structure/flora/tree/pine/xmas/presents,
```

Comment:
HEY

Why the hell didn't you mention the removal of the christmas tree in the PR body?! It's, like, one of the most interesting parts of Snowdin for people who aren't interesting in dungeon diving for gamer loot.

Reply:
okay this part is lame please fix